AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -185,7 +185,7 @@ robotikforum.net,kinogoo.cc,boyfriendtv.com,newsnow.co.uk,espinof.com,deleted.io
 !
 !--- Admiral ---
 ! can be closed
-superherohype.com,news24.com,pgatour.com,wral.com,gamepur.com,boredpanda.com,fin24.com,geekzone.co.nz,upi.com,thewindowsclub.com,androidcure.com,secondnexus.com,health24.com,wokesloth.com,sport24.co.za,onmsft.com,cultofmac.com,washingtontimes.com#%#AG_abortInlineScript(/admiral/, 'document.createElement');
+wrestlinginc.com,superherohype.com,news24.com,pgatour.com,wral.com,gamepur.com,boredpanda.com,fin24.com,geekzone.co.nz,upi.com,thewindowsclub.com,androidcure.com,secondnexus.com,health24.com,wokesloth.com,sport24.co.za,onmsft.com,cultofmac.com,washingtontimes.com#%#AG_abortInlineScript(/admiral/, 'document.createElement');
```

Comment:
block Admiral domain in Tracking Protection
`||cloisteredcord.com^$third-party`

Reply:
done